releasenotes/notes/fix-forbiddenfruit-error-ffbb174ade1c6d57.yaml

```
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Remove forbiddenfruit as dependency and rollback wrapt changes where forbiddenfruit was called.
```

```suggestion
    Remove ``forbiddenfruit`` as dependency and rollback wrapt changes where ``forbiddenfruit`` was called.
```

You might get spelling issues for this? (ignore if not)